infor-design/enterprise
# 4877 - Improve Horizontal/Header Tabs focus indicator placement

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR fixes a couple issues related to Header Tabs: 
- In RTL Mode, the placement of the focus indicator within the tab container was way off.  The placement has been improved.
- In RTL Mode, the faded edges were not facing the correct direction, and were not responding correctly to the container's scrolling (if it's possible to scroll left, the left edge should appear "faded" and visually cut off the contents, and vice-versa).  This has been fixed.
- A regression was discovered while testing in Safari where it wasn't possible to navigate the tabs with the arrow keys.  This has been fixed.

**Related github/jira issue (required)**:
- Closes #4877 
- Related to #4763 (caused the regression)

**Steps necessary to review your pull request (required)**:
_To test #4877:_
- Pull, build, run
- Open http://localhost:4000/components/tabs-header/example-index.html?locale=ar-SA
- Click on any tab, then navigate with the arrow keys.  The white box (focus indicator) should appear to be placed correctly over top of the tabs receiving focus.
- Press enter to select some tabs, and ensure the white line (selected indicator) closely lines up with the focus indicator.
- Check the edges of the tab container while scrolling and make sure the faded blue edges don't appear to be facing the wrong direction.
- Also check in-page tabs on RTL mode (as well as standard mode):
  - http://localhost:4000/components/tabs/example-dismissible-tabs.html
  - http://localhost:4000/components/tabs/example-dismissible-tabs.html?locale=ar-SA

_To test the regression:_
- Open http://localhost:4000/components/tabs/example-dismissible-tabs.html on an iPad (or simulator) running iOS 13+
- Try to dismiss the tabs by clicking the "X".  They should be removed as expected.

**Included in this Pull Request**:
